test/corectl_integration_test.go

```diff
@@ -168,7 +168,7 @@ func TestUsingJwt(t *testing.T) {
 	p4 := p.WithParams(toolkit.Params{Engine: *toolkit.EngineJwtIP, Config: "test/projects/using-jwts/corectl.yml"})
 
 	p1.ExpectOK().ExpectIncludes("Connected without app to").Run("status")
-	p2.ExpectErrorIncludes("Please check the --engine parameter or your config file.").Run("status")
+	p2.ExpectErrorIncludes("--engine", "config", "engine is running").Run("status")
```

Comment:
Has the expected outcome changed for this test case?

Reply:
Yes, I updated the error message